.idea/misc.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6.0 (C:\Python36-32\python.exe)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6.0 (C:\Users\marti\AppData\Local\Programs\Python\Python36-32\python.exe)" project-jdk-type="Python SDK" />
```

Comment:
Vi må være flinkere til å ikke inkludere IDEA filer.. Det vil skape problemer for hver gang man puller om ikke.